### PR TITLE
Fix/ally sept 2020

### PR DESCRIPTION
**Pull request by freesteph**
# Description
Attempt to address all the manually reported issues from our Admin portal.

# Review
- Read commit log;
- Try branch locally.

### CODE REVIEW

File: app/views/overview/index.html.erb [hunk 1]
```diff
@@ -1,13 +1,15 @@
 <% unless @current_org_signed_mou %>
-<div class="govuk-grid-row">
-  <div class="govuk-warning-text" id="mou-warning">
-    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-    <strong class="govuk-warning-text__text">
-      <span class="govuk-warning-text__assistive">Warning</span>
-      You must sign the GovWifi <%= link_to "memorandum of understanding", mou_index_path, class: "govuk-link govuk-link--no-visited-state" %> to use this service.
-    </strong>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
```
Comment (robinmitra) [inline app/views/overview/index.html.erb:3]: @freesteph Which guideline does this relate to?

Comment (freesteph) [reply app/views/overview/index.html.erb:3]: I'm not sure, I just fixed it as a I was tidying up the page and it looked bad on mobile.

File: app/views/overview/index.html.erb [hunk 2]
```diff
@@ -1,43 +1,45 @@
...
```
Comment (robinmitra) [inline app/views/overview/index.html.erb:3]:
```suggestion
    <div class="govuk-grid-column-full">
```
You're right. But on desktop, it unnecessary wraps (and I remember us fixing this a while back). The suggestion will fix this.

File: app/assets/stylesheets/components/overview.scss [hunk 3]
```diff
@@ -1,17 +1,5 @@
...
-  color: $colour-white;
-}
-
-.summary-link-colour:visited {
-  color: $colour-white;
-}
-
-.summary-link-colour:hover {
-  color: $colour-white;
+  border: 3px solid $colour-blue;
```
Comment (robinmitra) [inline app/assets/stylesheets/components/overview.scss:2]: This seems like a significant change visually. Do you think we can make it work like this?
Kapture 2020-09-17 at 9 55 46

File: app/views/overview/index.html.erb (hunk 2 above)
Comment (freesteph) [reply app/views/overview/index.html.erb:3]: > For most types of page, we recommend using either a ‘two-thirds’ or a ‘two-thirds / one third layout’. That stops lines of text getting so long that the page becomes difficult to read on desktop devices.

https://design-system.service.gov.uk/styles/layout/

File: app/assets/stylesheets/components/overview.scss (hunk 3 above)
Comment (freesteph) [reply app/assets/stylesheets/components/overview.scss:2]: think you might be caching old CSS, the new version is this: 

file:///home/steph/Pictures/Screenshot%20from%202020-09-17%2010-25-31.png

Comment (freesteph) [reply app/assets/stylesheets/components/overview.scss:2]: Screenshot from 2020-09-17 10-25-31

Comment (robinmitra) [reply app/assets/stylesheets/components/overview.scss:2]: Yes that's what I meant when I said the change looks significantly different visually. The gif I attached keeps the look how it used to be while still keeping the contrast acceptable.

Having said that, I just went back to the audit doc and the issue was actually having white text over yellow background, when links in those boxes are focused. So the only change we need to make is to change the link text to black when focused. I've checked the contrast against the blue background and it seems acceptable for WCAG 2.1A.

File: app/views/overview/index.html.erb (hunk 2 above)
Comment (robinmitra) [reply app/views/overview/index.html.erb:3]: It's already in a three-quarters column, and restricting it further to two-thirds effectively makes it one-half, hence I found it looking a bit odd. This is non-blocking so feel free to disregard if you disagree.

**Review by robinmitra — APPROVED** [review]
🎉

**Pull request merged**